fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PrivilegeCheckerV2.java

```diff
@@ -275,7 +278,8 @@ static String getTableNameByRoutineLoadLabel(ConnectContext context,
     static void checkDbAction(ConnectContext context, String catalogName, String dbName,
                               PrivilegeType.DbAction action) {
         if (!CatalogMgr.isInternalCatalog(catalogName)) {
-            throw new SemanticException(EXTERNAL_CATALOG_NOT_SUPPORT_ERR_MSG);
+            // throw new SemanticException(EXTERNAL_CATALOG_NOT_SUPPORT_ERR_MSG);
```

Comment:
Can this comment and some of the following ones be removed?

Reply:
This will be removed soon after external catalog privilege check is supported, so just keep it for a while.